demo/dev/index.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { render } from 'react-dom';
+
+import ReactProjectionGrid from '../../src/index';
```

Comment:
You can use webpack resolve config to differentiate the way you reference the source.

Reply:
Not necessary to duplicate the code